experimental/dds/tree2/src/shared-tree/sharedTree.ts

```diff
@@ -220,9 +226,25 @@ export enum ForestType {
 	Optimized = 1,
 }
 
+/**
+ * Used to distinguish the type of encoding method applied to the tree summary.
+ * @alpha
+ */
+export enum SummaryEncodeType {
+	/**
+	 * Optimized for encoded size and encoding performance.
+	 */
+	Compressed = 0,
+	/**
+	 * Simple uncompressed encoding, which does not use polymorphism, identifier deduplication or schema based compression.
+	 */
+	Uncompressed = 1,
```

Comment:
This enum could be renamed to be general enough to use when configuring op encoding as well.
Also some pedantic clarifications and tweaks to emphasis:

```suggestion
/**
 * Selects which heuristics to use when encoding tree content.
 * All encoding options here are compatible with the same decoder:
 * the selection here does not impact compatibility.
 * @alpha
 */
export enum TreeCompressionStratagy {
	/**
	 * Optimized for encoded size.
	 * Use this in production to reduce bandwidth and storage use.
	 */
	Compressed = 0,
	/**
	 * Optimized for human readability.
	 * Use this when debugging or testing and needing to inspect encoded tree content.
	 */
	Uncompressed = 1,
```

Reply:
updated enum with suggested change